File: Src/Icm.Core.Tests/Collections extensions/TextWriterExtensionsTest.vb
Imports System.IO
Imports Icm.IO

<TestFixture(), Category("Icm")>
Public Class TextWriterExtensionsTest

    <Test()>
    Public Sub WriteUnderlineTest()
        Dim tw As TextWriter = New StringWriter
        Dim s = "hola"
        tw.WriteUnderline(s)

        Dim actual = tw.ToString

        Dim expected = "hola" & vbCrLf & "----" & vbCrLf

        Assert.That(actual, [Is].EqualTo(expected))
    End Sub
End Class

File: Src/Icm.Core.Tests/Collections extensions/TextWriterExtensionsTest.cs

using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.IO;
using Icm;
using Icm.IO;
using NUnit.Framework;

[TestFixture(), Category("Icm")]
public class TextWriterExtensionsTest
{

	[Test()]
	public void WriteUnderlineTest()
	{
		TextWriter tw = new StringWriter();
		dynamic s = "hola";
		tw.WriteUnderline(s);

		dynamic actual = tw.ToString;

		dynamic expected = "hola" + Constants.vbCrLf + "----" + Constants.vbCrLf;

		Assert.That(actual, Is.EqualTo(expected));
	}
}

//=======================================================
//Service provided by Telerik (www.telerik.com)
//Conversion powered by NRefactory.
//Twitter: @telerik
//Facebook: facebook.com/telerik
//=======================================================
